### ISSUE TITLE
Next steps after changing configuration.yaml? Also clarify what ID is on thermostat

### ISSUE THREAD

**Issue by tmanpdx** [home-assistant/home-assistant.github.io#17289]
### Feedback

After modifying the configuration.yaml to include the climate section as outlined, what is the next step? Eg, how do I get this to show up in the configuration/integration section? 

Also - the doc says you can include the thermostat ID. Is this the MAC ID or the MAC CRC of the thermostat or something different? 

### URL

https://www.home-assistant.io/integrations/honeywell/

### Version

2021.3.4

**Comment by frenck** [comment]
Setting:

```yaml
climate:
# Example configuration.yaml entry
  - platform: honeywell
    username: YOUR_USERNAME
    password: YOUR_PASSWORD
```

Should be enough. Please note, you need to restart Home Assistant after adding integration to YAML configuration in order to make it pick up.

The configuration -> integration screen is only to manage integrations configured via the UI. This integration will not show up there.

**Comment by CTJohnK** [comment]
I added this to my configuration.yaml file and also expected to see it appear in the integration screen.  I do see it in the entity screen under 'climate'.

However, if I add 'thermostat' with the unit's four digit id (FxxD) or try it's twelve digit MAC address (00xxyyzzwwF3) the device no longer appears in my entity list.

I see nothing in the System Log regarding this.

**Comment by frenck** [comment]
> However, if I add 'thermostat' with the unit's for digit id (FxxD) or try it's MAC address (00xxyyzzwwF3) the device no longer appears in my entity list.

The documentation has 0 mentions of mac addressed or the units digits, not sure where you found that or why you think that. This is a cloud-based integration, that just needs a username and password 🤷

**Comment by CTJohnK** [comment]
Both the ID and MAC are available from the thermostat's screen if you go looking for it.  The ID is used when you pair the unit to the mobile app for example.

As for adding the 'thermostat' tag to the configuration.yaml, it is noted in the documentation, so I tried it.

I do see everything I would expect in HA, so this is working.  Thank you.

I did notice the Fan Mode reports "On", "Auto" or "diffuse" rather than "On", "Auto" or "Off"
